bin/i18n-codeorg/lib/merge-all-locales.rb

```diff
@@ -24,8 +24,8 @@ def merge_translation_tree(en_translation, new_translation, prev_translation)
       if en_translation.has_key?(key) && prev_translation.has_key?(key)
         new_translation[key] =
           merge_translation_tree(en_translation[key],
-                                 new_translation[key],
-                                 prev_translation[key])
+            new_translation[key],
+            prev_translation[key])
```

Comment:
We should adopt a standard as to where the closing parenthesis goes. Do you prefer
```
new_translation[key] =
  merge_translation_tree(en_translation[key],
    new_translation[key],
    prev_translation[key])
```
or
```
new_translation[key] =
  merge_translation_tree(en_translation[key],
    new_translation[key],
    prev_translation[key]
  )
```